dim_checks/moz-fx-data-shared-prod/telemetry/active_users_aggregates/dim_checks.yaml

```diff
@@ -0,0 +1,108 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  alerts_enabled:
+    enabled: true
+    notify:
+      channels:
+        - dim_test_run
+  tier: tier_1
+  partition_field: submission_date
+  dim_tests:
+    - type: table_row_count
+      params:
+        condition: ">= 1000"
+    - type: not_null
+      params:
+        columns:
+          - submission_date
+          - client_id
+          - first_seen_date
+          - first_seen_year
+          - app_name
+          - app_version
+          - channel
+          - segment
+          - attributed
+          - os
+          - os_version
+          - os_version_major
+          - os_version_minor
+          - dau
+          - wau
+          - mau
+          - new_profiles
+    - type: column_length
+      params:
+        columns:
+          - country
+        condition: "= 2"
+    - type: matches_regex
+      params:
+        columns:
+          - country
+        regex: ^[A-Z]{2}$  # example: GB
+    - type: value_in_set
+      params:
+        columns:
+          - channel
+        expected_values:
+          - nightly
+          - aurora
+          - release
+          - Other
+          - beta
+          - esr
+    - type: value_in_set
+      params:
+        columns:
+          - segment
+        expected_values:
+          - casual_user
+          - regular_user
+          - infrequent_user
+          - other
+          - core_user
+    - type: value_in_set
+      params:
+        columns:
+          - app_name
+        expected_values:
```

Comment:
Some missing values? See [the full list here](https://github.com/mozilla/dim/blob/2fd107e4349d8fc70ce0538a905cd2ac9430a188/dim_checks/moz-fx-data-shared-prod/telemetry/unified_metrics/dim_checks.yaml#L74).